python/ola/PidStore.py

```diff
@@ -116,6 +116,26 @@ def RequestSupported(self, command_class):
     """Check if this PID allows a command class."""
     return self._requests.get(command_class) is not None
 
+  def GetRequest(self, command_class):
+    return self._requests.get(command_class)
+
+  def GetRequestField(self, command_class, field_name):
+    field = filter(lambda field: field.name == field_name,
+                   self._requests.get(command_class).GetAtoms())
+    return field[0] if (len(field) > 0) else None
```

Comment:
I'd change the variable to fields and do

return fields[0] if not fields else None